front/app/translations/admin/ar-SA.json

```diff
@@ -1803,12 +1803,8 @@
   "app.containers.admin.import.importStepOne": "1. تنزيل وتعبئة القالب",
   "app.containers.admin.import.importStepTwo": "2. تحميل ملفك المكتمل",
   "app.containers.admin.import.successMessage": "تم استيراد الملف بنجاح!",
-  "app.containers.admin.inviteUsersWithSeatsModal.additionalAdminAndManagerSeats": "{adminSeats, plural, one {1 مقعد مشرف إضافي} other {# مقاعد إضافية للمشرف}} و {managerSeats, plural, one {1 مقعد مدير إضافي} other {# مقعد مدير إضافي}}",
-  "app.containers.admin.inviteUsersWithSeatsModal.additionalAdminSeats": "{seats, plural, one {1 مقعد مشرف إضافي} other {# مقاعد إضافية للمشرف}}",
-  "app.containers.admin.inviteUsersWithSeatsModal.additionalManagerSeats": "{seats, plural, one {1 مقعد مدير إضافي} other {# مقعد مدير إضافي}}",
```

Comment:
I don't know why this PR has picked up these changes, but I don't think it's harmful.

Also happened in [this PR](https://github.com/CitizenLabDotCo/citizenlab/pull/4617).